dotCMS/src/main/webapp/WEB-INF/velocity/static/htmlpage_assets/redirect_custom_field.vtl

```diff
@@ -4,23 +4,23 @@
 		pageSelector.show();
 	}
 	function redirectPageSelected(page) {
-		dijit.byId('text8').set('value', page.pageURI);
+		dijit.byId('redirectURLBox').set('value', page.pageURI);
 	}
 
 	function updateRedirectValue() {
-		dojo.byId('redirecturl').value=dijit.byId('text8').get('value');
+		dojo.byId('redirecturl').value=dijit.byId('redirectURLBox').get('value');
 	}
 
 	dojo.ready(function() {
-		dijit.byId('text8').set('value', dojo.byId('redirecturl').value);
-		dojo.connect(dijit.byId('text8'),"onchange",updateRedirectValue);
+		dijit.byId('redirectURLBox').set('value', dojo.byId('redirecturl').value);
+		dojo.connect(dijit.byId('redirectURLBox'),"onChange",updateRedirectValue);
 	});
 
 </script>
 
 
 <input type="text" dojoType="dijit.form.TextBox" style="width:275px;" 
-       id="text8" name="text8" />
+       id="redirectURLBox" name="redirectURLBox" />
```

Comment:
could redirectURLBox be a constant?(I don't how constant work in vtl), this way is not need to repeat the value in every place where it is use